sequence/logout.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+title User Logout
+actor User as user
+participant SDK as sdk
+participant Webview as wv
+participant "Authorization\nServer" as as
+participant "Web\nServer" as ws
+participant "User\nDatabase" as db
+actor Admin as admin
+
+admin -> db: logout = now
+user -> sdk: Clicks Widget
+activate sdk
+  sdk -> wv: Open Webview
+  activate wv
+    wv -> ws: Fetch Homepage
+    ws --> wv: Homepage
+    wv -> as: POST /token\ngrant_type=cookie\n&client_id=webview_version
+    activate as
+    as -> db: Find User
+    db --> as: User Data\nlogout later than token
+    as --> wv: {loggedIn=false,nonce=<nonce>}
+    deactivate as
+    wv --> user: Homepage, User Logged Out
+  deactivate wv
+deactivate sdk
+break 
+ sdk -> as: client_id=sdk_version\n&refresh_token=refresh_token
+ activate sdk
+    activate as
+  as -> db: Find User
+  db --> as: User Data\nlogout later than token
+  as --> sdk: {error=access_denied}
+    deactivate as
+deactivate sdk
+@enduml
```

Comment:
Great to see this plantuml diagrams added to the source. Not a blocker but we should add an image of the rendered diagrams to the source like our other repos.